src/ucp/api/ucp.h

```diff
@@ -277,6 +278,13 @@ enum ucp_ep_params_flags_field {
                                                            send to a particular
                                                            remote endpoint, for
                                                            example stream */
+    UCP_EP_PARAMS_FLAGS_SEND_WORKER_ID = UCS_BIT(2)   /**< Send endpoint's worker id
+                                                           when connecting to remote
+                                                           socket address as part of
+                                                           connection request payload
+                                                           and can be obtained from
```

Comment:
"and can" is ambiguous.  need to specify what can be obtained from....and rest of sentence doesn't flow